Quick-Start/Prerequisite.md

```diff
@@ -60,7 +60,7 @@ PostgreSQL supports various target languages for extension development. Here is
 
 ## C:
```

Comment:
I suggest starting with SQL, then PL/pgSQL, and then C and the other languages. Start with the simplest first.

Reply:
I have just listed these out to inform the user about the various target languages which are supported for extension development.
I'll only be working on a selected few languages in this tutorial.

Reply:
Sure, I'm just suggesting to list them in order of complexity to use.

Reply:
Oh !
yup, I'll do that.